ConfigurationSystem/Client/Utilities.py

```diff
@@ -486,11 +486,9 @@ def getDBParameters( fullname, defaultHost = 'localhost',
   'db' and 'queueSize'
   """
 
-  fullname = fullname
```

Comment:
Yes, I actually meant it for the others, sorry.
But you're right, I overlooked it, this code is actually buggy and SHOULD use the one given as params if it cannot find it in the two CS places